platform/values.yaml

```diff
@@ -1,85 +1,132 @@
+redis:
+  fullnameOverride: redis
+  usePassword: false
+  cluster:
+    enabled: false
+openldap:
+  fullnameOverride: ldap-openldap
+  enabled: true
+  adminPassword: ''
+  env:
+    LDAP_BASE_DN: dc=example,dc=org
+  customLdifFiles:
+    init.ldif : |
+      dn: ou=orgs,dc=example,dc=org
+      objectClass: top
+      objectClass: organizationalUnit
+      ou: orgs
 andi:
   enabled: true
+  image:
+    tag: 0.6.0
   redis:
-    host: platform-redis-master
+    host: redis
   strimzi:
     kafka:
       brokers: streaming-service-kafka-bootstrap:9092
+  service:
+    type: ClusterIP
 discovery-api:
-  enabled: true
-  replicaCount: 1
-  redis:
-    host: platform-redis-master
-  ldap:
-    host: ldap
-discovery-streams:
-  enabled: false
-discovery-ui:
-  enabled: true
-flair:
-  enabled: true
-  redis:
-    host: platform-redis-master
-  strimzi:
-    kafka:
-      brokers: streaming-service-kafka-bootstrap:9092
-forklift:
-  enabled: true
+  image:
+    tag: 0.16.2
   redis:
-    host: platform-redis-master
+    host: redis
+  presto:
+    url: http://kdp-kubernetes-data-platform-presto:8080
+  postgres:
+    host: scosdev.postgres.database.azure.com
   strimzi:
     kafka:
       brokers: streaming-service-kafka-bootstrap:9092
-kdp:
+  s3:
+    hostedFileBucket: hosted-bucket
```

Comment:
We shouldn't default this to anything and make sure to call it out as a required parameter.